client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/ClientListenerWithDslFilterObjectStorageTest.java

```diff
@@ -81,8 +81,7 @@ protected ConfigurationBuilder getConfigurationBuilder() {
       cfgBuilder.encoding().key().mediaType(MediaType.APPLICATION_OBJECT_TYPE);
       cfgBuilder.encoding().value().mediaType(MediaType.APPLICATION_OBJECT_TYPE);
       cfgBuilder.indexing().enable()
-            .addProperty("default.directory_provider", "local-heap")
-            .addProperty("lucene_version", "LUCENE_CURRENT");
```

Comment:
I think you don't have to remove this, just replace `LUCENE_CURRENT` with `LATEST`.

Reply:
Though this property is indeed not necessary (LATEST is the default)

Reply:
Shall I replace the old `LUCENE_CURRENT` with the `LATEST`? I removed them since in general I prefer have less code than more :P and in this case `LATEST` is the default one

Reply:
I think it was there to avoid the warning, maybe? I just mentioned this because I was surprised by this seemingly unnecessary change.

Anyway, it's probably not worth your time. Let's dropt his, sorry for the noise.

Reply:
@yrodiere No noise at all, thanks. I asked myself the same question when I did this change.